packages/thirdweb/src/auth/verify-hash.ts

```diff
@@ -163,10 +162,14 @@ async function verifyEip1271Signature({
   signature: Hex;
   contract: ThirdwebContract;
 }): Promise<boolean> {
-  const result = await isValidSignature({
-    hash,
-    signature,
-    contract,
-  });
-  return result === EIP_1271_MAGIC_VALUE;
+  try {
+    const result = await isValidSignature({
+      hash,
+      signature,
+      contract,
+    });
+    return result === EIP_1271_MAGIC_VALUE;
+  } catch {
```

Comment:
Generally really want to log something for the dev to understand why the sig was not valid, same with the catch above